### PR TITLE
Fix link to CI workflow from CI badge

### PR DESCRIPTION
**Pull request by mbrukman**
The default link for a workflow badge points to the image itself.
Update link to point to the Actions tab on GitHub with the list of runs.
[skip ci] since we are not modifying any code that would impact CI.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

**Comment by mbrukman** [comment]
Sorry, I would need to have lawyers review your CLA before signing it, which isn't worth it for a trivial change such as this one.

FWIW, you should clearly state that contributions require signing a CLA in [`CONTRIBUTING.md`](https://github.com/zenml-io/zenml/blob/c41dd353acbb4ff32fc39292e99a0a6541f80203/CONTRIBUTING.md) which it currently doesn't — I searched again for mentions of either "CLA" or "Contributor License Agreement" and it's not there.

**Comment by htahir1** [comment]
So sorry that you had to close this PR, and felt blindsided by the CLA. I have remedied  this with a clear reference in the CONTRIBUTING.md as per your suggestion (#536). The CLA is just to protect the core contributors as much as it is to everyone contributing to ZenML, and does not mean to be jargon-y. I will bring up with the core team how we can make it easier to understand what the contributing licence is in plain english so that you and others in the future do not get dissuaded to contribute.  So sorry for the inconvenience!
